base/src/expressions/types.rs

```diff
@@ -30,6 +30,7 @@ pub struct CellReference {
 #[derive(Clone)]
 pub struct CellReferenceRC {
     pub sheet: String,
+    // 1-based index
```

Comment:
Again, rows and columns are *always* 1-based and sheets 0-based. I think we should only document it once.